### PR TITLE
Legacy torque accounting lines

### PR DESCRIPTION
**Pull request by itkovian**
Torque parsing issues addressed:
- resources_used.cput can be given as HHHH:MM:SS instead of just seconds.
- deal with negative exit codes
- old torque had `;A;` entries (aborted jobs?)
- reduce the amount of data in the execHosts field by reducing it to a canonical form
- array job queue lines have `[]` in the job name
- epilogue and prologue fields in the Resource_List

Other goodies:
- Count the messages parsed
- Print a timestamp every 10K messages and indicate successes and failures

### CODE REVIEW

File: hnormalise.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:                hnormalise
-version:             0.4.6.0
+version:             0.4.7.0
 synopsis:            Log message normalisation tool producing structured JSON messages
 description:         Log message normalisation tool producing structured JSON messages
 homepage:            https://github.com/itkovian/hnormalise#readme
```
Comment (stdweird) [inline hnormalise.cabal]: next time you choose a license for a project, can you discuss this first in  a meeting. also, please change the copyright, it belongs to ugent.

Comment (itkovian) [reply hnormalise.cabal]: Since I've worked on it during work hours, I guess it does now

Comment (stdweird) [reply hnormalise.cabal]: feel free to add yourself as author though 😄

**Comment by itkovian** [comment]
Fixed #6

**Comment by boegel** [comment]
@itkovian This PR is quickly becoming too big to review properly, and it's also too much of a moving target...

Can we split this up so it's feasible to review? If you make any further enhancements, please fork off a new branch to keep things manageable?

**Comment by itkovian** [comment]
Will not be making further enhancements. please review.

**Pull request merged**